src/postBackEnd.js

```diff
@@ -0,0 +1,173 @@
+import * as firebase from 'firebase';
+import { db } from './index.js';
+import { currentUserObj, userExist } from './signIn.js';
+
+
+///////////////////////////////////////////////////////
+//Post gets and sets
+///////////////////////////////////////////////////////
+
+export function getAddress(post_id){
+  const collect = db.collection('post')//get wanted collection
+  query = collect.where('post_id', '==', post_id)
+  query.get().then(queriedDocs => {
+      if (queriedDocs.empty == false) {
+          queriedDocs.forEach(singleDoc => {
+              return singleDoc.data().address;
+          })
+      } else {
+          console.log("No docs match");
+      }
+  });
+}
+
+export function setAddress(post_id, newAddress){
+  db.collection('post').doc(post_doc).update({
+      addres: newAddress
+  }).catch(function (error) {
+      // The document probably doesn't exist.
+      console.error("Error updating document: ", error);
+      return false
+  });
+  return true
+}
+
+// Thinking it might be better to simply do one search for every comment
+// item with a given post_id than a serach for a specific comment id for
+// every value in an array of comments.  Will talk with Branden about this
+
+// export function getComments(post_id){
+//   const collect = db.collection('post')//get wanted collection
+//   query = collect.where('post_id', '==', post_id)
+//   query.get().then(queriedDocs => {
+//       if (queriedDocs.empty == false) {
+//           queriedDocs.forEach(singleDoc => {
+//               return singleDoc.data().comments;

```

Comment:
Your only returning a single comment for the given post. You will need to return a list of all the comments.

Reply:
The comments get function I commented out because we had decided to instead to a search of a post's comments in the comments table using a post_id, can be deleted but hadn't gotten around to it after our discussion.